UnityProject/Assets/Scripts/Player/PlayerFlashEffects.cs

```diff
@@ -2,14 +2,20 @@
 using Mirror;
 using CameraEffects;
 using Messages.Server;
+using Items.Implants.Organs;
+using System.Collections.Generic;
 
 namespace Player
 {
 	public class PlayerFlashEffects : NetworkBehaviour
 	{
+		public List<WeldingShieldImplant> WeldingShieldImplants = new List<WeldingShieldImplant>();
+
 		[Server]
 		public void ServerSendMessageToClient(GameObject client, float newValue)
 		{
+			if (WeldingShieldImplants.Count > 0) return;
```

Comment:
do we have a welding effect I thought this was for shooting guns? Muzzle light flash

Reply:
We do not having a welding effect yet, but this implant sound also work against flashers

Reply:
cool